docs/fub_curta.md

```diff
@@ -27,7 +33,3 @@ If computing time on Curta has contributed to a publication or the completion of
 The current list of publications is available [here](https://www.fu-berlin.de/en/sites/high-performance-computing/Forschungsergebnisse).
 
 Please also acknowledge the use of our service in any publications you produce, using the following [reporting guidelines](https://www.fu-berlin.de/en/sites/high-performance-computing/FAQ/Publikationen), including the DOI [10.17169/refubium-26754](http://dx.doi.org/10.17169/refubium-26754).
-
-## Acknowledgments
-
-This configuration was refined with Dr. Loris Bennett's help [@tardigradus](https://github.com/tardigradus) of the HPC Team from the FUB ZEDAT's Scientific Computing Unit.
```

Comment:
Any reason why this is being removed?

Reply:
Hi James. Thanks for approving the changes. I also felt a bit conflicted about this initially, but eventually decided to go forward with it, my reasoning behind it being that, despite the HPC team having answered some basic memory and CPU questions about the cluster in the initial stages (which is why I included this part at first), ultimately the creation and improvement of the profile itself and figuring out the best options and setup was done pretty much all by me on my own time and initiative. So that's why it seemed right to me. I hope that's alright?

Reply:
Yup that's fine! It's not standard for docs anyway :)

Merge away!